lib/src/participant/local_participant.dart

```diff
@@ -296,13 +296,16 @@ class LocalParticipant extends Participant<LocalTrackPublication> {
       return publication;
     } else if (enabled) {
       if (source == TrackSource.camera) {
-        final track = await LocalVideoTrack.createCameraTrack();
+        final track = await LocalVideoTrack.createCameraTrack(
+            room.roomOptions?.defaultVideoCaptureOptions);
         return await publishVideoTrack(track);
       } else if (source == TrackSource.microphone) {
-        final track = await LocalAudioTrack.create();
+        final track = await LocalAudioTrack.create(
+            room.roomOptions?.defaultAudioCaptureOptions);
         return await publishAudioTrack(track);
       } else if (source == TrackSource.screenShareVideo) {
-        final track = await LocalVideoTrack.createScreenShareTrack();
+        final track = await LocalVideoTrack.createScreenShareTrack(
+            room.roomOptions?.defaultVideoCaptureOptions);
```

Comment:
Are these the same in Flutter case? I am guessing we are doing Simulcast for camera and non-Simulcast for screen share. Can Flutter do that?

Reply:
Makes sense to separate defaults for cam and screen share 😁